catch/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+# Just point the user at our vendored catch. Nothing else to do here
```

Comment:
I see why this makes sense (we may eventually have vendored libraries that we don't plan to export), and the vendor dir pattern is a useful identifier. So this is fine.